src/object/srv_ec_aggregate.c

```diff
@@ -2319,7 +2321,9 @@ agg_iterate_pre_cb(daos_handle_t ih, vos_iter_entry_t *entry,
 		if (!(*acts & VOS_ITER_CB_SKIP))
 			agg_reset_pos(type, agg_entry);
 		if (ec_aggregate_yield(agg_param)) {
+			/* post_cb may not be called, so let's process current stripe */
 			D_DEBUG(DB_EPC, "EC aggregation aborted\n");
+			agg_process_curr_stripe(ih, agg_param, entry, agg_entry, acts);
```

Comment:
I am not sure this is safe to do after re-probe.   Is there not an alternative where we simply clean-up what we've allocated?   Or is it too late at this point.    I believe the nested iterator will do a dbtree_iter_fetch on the entry in question and I'm not sure it's valid.  If more I/O happened, the tree pointers may have moved.  The alternative is to add a callback for "exit" but that probably isn't the right answer for 2.0.